external/CMakeLists.txt

```
@@ -29,16 +29,15 @@ endif()
 # --------------------------------------------------------------------------------------------------
 # zenoh
 
-set(GIT_TAG "0d0a6d8d1488a71224c3bf6929714036945e7947")
+set(ZENOH_VERSION "0.11.0.3")
 add_cmake_dependency(
-  NAME zenohc URL "https://github.com/eclipse-zenoh/zenoh-c/archive/${GIT_TAG}.zip"
-  CMAKE_ARGS -DZENOHC_CARGO_CHANNEL=1.76.0
+  NAME zenohc URL "https://github.com/eclipse-zenoh/zenoh-c/archive/refs/tags/${ZENOH_VERSION}.zip"
```

given that we now use a release candidate we can do:
```suggestion
  NAME zenohc 
  GIT_REPOSITORY "https://github.com/eclipse-zenoh/zenoh-c"
  GIT_TAG ${ZENOH_VERSION}
```

and same for `zenohcxx`